test/index.js

```diff
@@ -0,0 +1,176 @@
+const expect = require('expect');
+const {createRobot} = require('probot');
+const plugin = require('..');
+const commentEvent = require('./fixtures/issue_comment.created');
+
+describe('snooze', () => {
+  let robot;
+  let github;
+
+  beforeEach(() => {
+    robot = createRobot();
+
+    // Load the plugin
+    plugin(robot);
```

Comment:
Ahh, yeah, `probot-visitor` will authenticate and start fetching installations when it loads. If you move this this _after_ mocking out `robot.auth` below, then you should be able to also mock out the API requests that `visitor` uses.